packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -1,595 +0,0 @@
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{-fauxconcat foo bar bing}} {{unbound (-fauxconcat foo bar bing)}}'),
-    context: EmberObject.create({
-      foo: 'a',
-      bar: 'b',
-      bing: 'c'
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'abc abc', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.bar', 'X');
-  });
-
-  equal(view.$().text(), 'aXc abc', 'unbound helpers/properties stayed the same');
-});
-
-QUnit.test('should be able to render an unbound helper invocation for helpers with dependent keys', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{-capitalizeName person}} {{unbound (-capitalizeName person)}} {{-concatNames person}} {{unbound (-concatNames person)}}'),
-    context: EmberObject.create({
-      person: EmberObject.create({
-        firstName: 'shooby',
-        lastName:  'taylor'
-      })
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'SHOOBY SHOOBY shoobytaylor shoobytaylor', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.person.firstName', 'sally');
-  });
-
-  equal(view.$().text(), 'SALLY SHOOBY sallytaylor shoobytaylor', 'only bound values change');
-});
-
-QUnit.test('should be able to render an unbound helper invocation in #each helper', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile(
-      ['{{#each people as |person|}}',
-       '{{-capitalize person.firstName}} {{unbound (-capitalize person.firstName)}}',
-       '{{/each}}'].join('')),
-    context: {
-      people: emberA([
-        {
-          firstName: 'shooby',
-          lastName:  'taylor'
-        },
-        {
-          firstName: 'cindy',
-          lastName:  'taylor'
-        }
-      ])
-    }
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'SHOOBY SHOOBYCINDY CINDY', 'unbound rendered correctly');
-});
-
-QUnit.test('should be able to render an unbound helper invocation with bound hash options', function() {
-  owner.register('helper:-repeat', helper(function([value]) {
-    return [].slice.call(arguments, 0, -1).join('');
-  }));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{-capitalizeName person}} {{unbound (-capitalizeName person)}} {{-concatNames person}} {{unbound (-concatNames person)}}'),
-    context: EmberObject.create({
-      person: EmberObject.create({
-        firstName: 'shooby',
-        lastName:  'taylor'
-      })
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'SHOOBY SHOOBY shoobytaylor shoobytaylor', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.person.firstName', 'sally');
-  });
-
-  equal(view.$().text(), 'SALLY SHOOBY sallytaylor shoobytaylor', 'only bound values change');
-});
-
-QUnit.test('should be able to render bound form of a helper inside unbound form of same helper', function() {
```

Comment:
:heavy_check_mark: should be able to render bound form of a helper inside unbound form of same helper